pkg/gui/containers_panel.go

```diff
@@ -571,3 +571,26 @@ func (gui *Gui) handleContainersBulkCommand(g *gocui.Gui, v *gocui.View) error {
 
 	return gui.createBulkCommandMenu(bulkCommands, commandObject)
 }
+
+// Open first port in browser
+func (gui *Gui) handleContainersOpenInBrowserCommand(g *gocui.Gui, v *gocui.View) error {
+	container, err := gui.getSelectedContainer()
+	if err != nil {
+		return nil
+	}
+	// skip if no any ports
+	if len(container.Container.Ports) == 0 {
+		return nil
+	}
+	// skip if the first port is not published or it's a UDP port
+	port := container.Container.Ports[0]
+	if port.IP == "" || port.Type != "tcp" {
```

Comment:
the new HTTP/3 runs on top of QUIC and that's running on top of UDP so it's probably best to remove this type check.
This protocol is also already used in major websites like google so it's not anymore some proposal that someday will be used.

Reply:
Ok, then we may remove the check

Reply:
I removed the check, please merge